cypress/cypress.js

```diff
@@ -1,72 +1,87 @@
 /* eslint-disable import/no-extraneous-dependencies, no-console */
-const atob = require('atob');
-const { execSync } = require('child_process');
-const { post } = require('request').defaults({ jar: true });
-const { seedData } = require('./seed-data');
+const atob = require("atob");
+const { execSync } = require("child_process");
+const { post } = require("request").defaults({ jar: true });
+const { seedData } = require("./seed-data");
 
-const baseUrl = process.env.CYPRESS_baseUrl || 'http://localhost:5000';
+const baseUrl = process.env.CYPRESS_baseUrl || "http://localhost:5000";
 
 function seedDatabase(seedValues) {
   const request = seedValues.shift();
-  const data = request.type === 'form' ? { formData: request.data } : { json: request.data };
+  const data =
+    request.type === "form"
+      ? { formData: request.data }
+      : { json: request.data };
 
   post(baseUrl + request.route, data, (err, response) => {
     const result = response ? response.statusCode : err;
-    console.log('POST ' + request.route + ' - ' + result);
+    console.log("POST " + request.route + " - " + result);
     if (seedValues.length) {
       seedDatabase(seedValues);
     }
   });
+
+  // Make sure the admin user has the same API key on every execution
+  execSync(
+    "docker-compose -p cypress run postgres psql -h postgres -c \"update users set api_key = 'secret' where email ='admin@redash.io';\""
+  );
```

Comment:
This is the actual test. The rest is Prettier doing its thing.

Reply:
Nice! That was one of the solutions I thought about (the other one would require me to use Cypress Stubs).

My idea is to create a few Cypress tests in #3354, probably one for regenerating the API Key. If needed in there, I'll revisit this.

BTW, I thought the pattern was to use singlequoted :sweat_smile:

Reply:
> BTW, I thought the pattern was to use singlequoted 😅

🤔 you're correct. Something went wrong there. I'll fix.